packages/next/client/image.tsx

```diff
@@ -209,6 +218,24 @@ function defaultImageLoader(loaderProps: ImageLoaderProps) {
   )
 }
 
+function removePlaceholder(
+  element: HTMLImageElement | null,
+  placeholder: PlaceholderValue
+) {
+  const hasBlurryPlaceholder =
+    configPlaceholder === PlaceholderValue.BLURRY &&
+    placeholder === PlaceholderValue.BLURRY
+  if (hasBlurryPlaceholder && element) {
+    if (element.complete) {
```

Comment:
Might want to put a comment here to make it clear that we know that this case triggers on broken images too, and that we plan to implement a broken image fallback in a future update.

Reply:
Good idea.